### DISCUSSION TITLE
cluster name and project id for GKE

### DISCUSSION THREAD

**Discussion by phamngocsonls** [aquasecurity/trivy-operator#1202] [Development]
Hi, I want to install trivy operator for multi cluster in GKE.
With all metrics push to Prometheus, I need 2 info about cluster-name and project-id (can get from GKE api).
How can I add more info for metrics? Dev team can provide for me about what file I can custom? with example metric: "trivy_resource_configaudits"

**Comment by chen-keinan** [comment]
@phamngocsonls this is the [metrics collector](https://github.com/aquasecurity/trivy-operator/blob/main/pkg/metrics/collector.go)
